openstack/migrate-helpers/security_group_exporter.rb

```diff
@@ -0,0 +1,56 @@
+#
+# security_group_exporter exports all of the security groups and creates a script to re-import them (most likely into a new openstack deployment).
+#
+# Note: As of now, the script just ignores the `default` security group.
+#
+# Original Author: AJ Bahnken / ajvb
+#
+
+require 'json'
+
+# Create output folder
+todays_date = Time.now.strftime "%Y-%m-%d"
+system 'mkdir', "secgroup_exporter_#{todays_date}"
```

Comment:
If we're going to use Ruby for this, we should either use the FileUtils module (preferred) or the shell syntax.

FileUtils
```
require 'fileutils'

FileUtils.mkdir("secgroup_exporter_#{todays_date}")
```

Shelling out
```
`mkdir secgroup_exporter_#{todays_date}"`
```